docs/website/docs/index.md

```diff
@@ -23,20 +23,20 @@ IREE supports importing from a variety of ML frameworks:
 - [x] TensorFlow
 - [x] TensorFlow Lite
 - [x] JAX
-- [ ] PyTorch
-<!-- - [ ] ONNX -->
+- [ ] PyTorch (planned)
+<!-- - [ ] ONNX (planned) -->
```

Comment:
Is "hoped" a category (as a step below "planned") :)

Reply:
Sure :P uncommented and changed to "(hoped for)"